src/app/users-page/author/add-authors/add-authors.component.spec.ts

```diff
@@ -32,21 +47,36 @@ describe('AddAuthorsComponent', () => {
     fixture.detectChanges();
   });
 
-  // it('form valid when input is entered', () => {
-  //   const firstNameInput = authorComponent.registerFormControl.first_name;
-  //   firstNameInput.setValue('Delyan');
+  it('should make field required', () => {
```

Comment:
For consistency, I think you should write this type of test for all the fields that are required on this form